data-avro/src/test/java/com/linkedin/data/avro/TestSchemaTranslator.java

```diff
@@ -70,8 +70,220 @@ public void testTranslateDefaultBackwardsCompatibility()
     assertSame(DataToAvroSchemaTranslationOptions.DEFAULT_OPTIONAL_DEFAULT_MODE, OptionalDefaultMode.TRANSLATE_DEFAULT);
   }
 
+  @DataProvider
+  public Object[][] toAvroSchemaDataTestTypeRefAnnotationPropagationUnionWithAlias()
+  {
+    return new Object[][]
+        {
+            {
+                "record test {" +
```

Comment:
Suggest to put these test data in a file, and leave the path/file name here for the test function, and then read the file during test, this way, it is easy to see the test data clear and if we want to change it is also easy to change

Reply:
I believe new tests in Pegasus are already written using the conversion you mentioned

TBH in order to achieve that in this particular file, maybe it is same as we discussed today: SchemaTranslator need to be rewritten and gradually migrated all the functionality.

Reply:
I think what Brian suggested was using pdl file(path) as input. You can use TestUtil#dataSchemaFromPdlInputStream(getClass().getResourceAsStream(<fileName>)) to get the DataSchema. It will be better than writing pdl string in the test class. 
However, for your test case, you still need avroschema as expected result. You may still end up writing avro schema string.

Reply:
I understood what Brian meant, but that is really impractical since that file included a lot of (small) tests